src/Commands/StartRoadRunnerCommand.php

```diff
@@ -120,6 +119,20 @@ protected function writeServerStateFile(
         ]);
     }
 
+    /**
+     * Provide road runner config file path.
+     *
+     * @return string
+     */
+    protected function configPath()
+    {
+        if (! $this->option('config-path')) {
+            touch(base_path('.rr.yaml'));
+            return base_path('.rr.yaml');
+        }
+        return $this->option('config-path');
```

Comment:
Something like below you mean?

```suggestion
        $configPath =  $this->option('config-path');
        if (! $this->isAbsolutePath($configPath)) {
            return base_path($configPath);
        }
        return $configPath;
```

And have a new function isAbsolutePath()

```
>>> function isAbsolutePath($path) {
...     if($path === null || $path === '') throw new Exception("Empty path");
...     return $path[0] === DIRECTORY_SEPARATOR || preg_match('~\A[A-Z]:(?![^/\\\\])~i',$path) > 0;
... }
```